src/pymor/models/iosys.py

```diff
@@ -627,37 +627,75 @@ def to_abcde_files(self, files_basename):
         if E is not None:
             _mmwrite(Path(files_basename + '.E'), E)
 
-    def _compute(self, solution=False, output=False, solution_d_mu=False, output_d_mu=False,
-                 solution_error_estimate=False, output_error_estimate=False,
-                 output_d_mu_return_array=False, mu=None, **kwargs):
+    def compute(self, solution=False, output=False, mu=None, input=None):
```

Comment:
The `solve` method should be updated to respect these arguments.

Reply:
Actually this we would also need to update all other methods defined in `models.interface`... I think it is better to revert the changes and leave these arguments in `compute` and instead raise `NotImplementedError` if any of those are called.

Reply:
Yes, true, it's probably better that way, I agree.